xml/schema/Activity/Activity.xml

```diff
@@ -263,6 +263,11 @@
       <type>Select</type>
     </html>
   </field>
+  <index>
+    <name>UI_status_id</name>
```

Comment:
Seems like this naming standard has changed and the current preferred default prefix is index_*

Please update.